webclient/app/src/app/assets/themes/MainTheme.js

```diff
@@ -95,6 +97,7 @@ class MainTheme {
                 // Name of the slot
                 primary: {
                     backgroundImage: "linear-gradient(-60deg, rgb(255, 88, 88) 0%, rgb(240, 152, 25) 100%)",
+                    // eslint-disable-next-line
                     boxShadow: "0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 10px 10px 10px 0px rgb(0 0 0 / 12%)"
```

Comment:
Warum wird hier eigentlich der Schatten angepasst? Nur so aus Interesse.
Theoretisch könnten wir auch ein DefaultProp setzen, dass das Fab mit einer anderen ``elevation`` in Standard besetzt wird. Dann müssten wir uns nicht mit dem Styling rumschlagen.

Reply:
klingt gut